doc/method/method.md

```diff
@@ -6,173 +6,348 @@ The assembly process works, in outline, as follows:
 E.g. the GBIF taxonomy is downloaded from the GBIF web site, and then
 converted to the Open Tree 'interim taxonomy format' by a python
-script.
-
-Following ingest the following two normalizations are performed:
-
- 1. "Containers" - nodes in the tree that don't represent taxa - are
-    removed and replaced by flags (node annotations indicating edge
-    type); the most prominent being the pseudo-taxon "Incertae sedis"
+script.  Given the ITF files, the taxonomy can be 'loaded' by the
+assembly procedure.
+
+In two cases an imported source taxonomy is split into parts before
+processing continues, so that the parts can be positioned in different
+places in the align/merge order: Index Fungorum is split into high
+priority Fungi and lower priority everything-except-Fungi, and WoRMS
+is split into Malacostraca and everything-except-Malacostraca.
+
+After each source taxonomy is loaded, the following two normalizations
+are performed:
+
+ 1. Child taxa of "containers" in the source taxonomy are made to be
+    children of the parent container's parent.  "Containers" are
+    grouping nodes in the tree that don't represent taxa, for example
+    "incertae sedis" or "environmental samples" nodes.  The fact that
+    the child had been in a container is recorded as a flag on the
+    child node.
  1. Monotypic homonym removal - when taxon with name N has as its
-    only child another taxon with name N, remove one of the two
-
-Before alignment and merge, each source taxonomy gets patched
-individually.  It's always best to treat a problem as early as
-possible, so that its ill effects don't interfere with alignment of
-other taxonomies and with proper synthesis.  This is separate from the
-general patch phase that takes place at the end of taxonomy
-construction (below). A frequent kind of patch is to add a synonym or
-change a name so that a source taxon aligns with a taxon from an
-earlier or later source.  Patches that bring a source taxonomy into
-agreement with the skeleton also happen here.
-
-## Separation
+    only child another taxon with name N, the parent is removed.
+    This is done to avoid an ambiguity when later on a node with name
+    N needs to be matched.
+
+## Source patching
+
+Most source taxonomies have ad hoc patches that are applied at this
+point.  Capitalizations andmisspellings can be fixed at this point,
+and synonyms added (e.g. 'Stramenopiles' for 'Heterokonta'), to
+improve matches between taxonomies.  Other patches such as extinct or
+extant annotations or topology changes can be applied at this point.
+Generally it's best to handle taxonomic problems by correcting source
+taxonomies at this point, but if it is difficult or inconvenient to
+localize the problem to one or more sources, repair can be left until
+the final patch phase after all source taxonomies have been loaded and
+merged.
+
+Editing the source taxonomy presents a provenance tracking problem
+that has yet to be addressed.
```

Comment:
but we aren't directly editing source taxonomies, are we? Changing format and applying patches, yes, but these are scripted and reproducible.